### ISSUE TITLE
Require Readme as Markdown

### ISSUE THREAD

**Issue by sfisher** [CDL-Dryad/dryad-product-roadmap#1873]
This ticket is fairly straightforward for us and doesn't have some of the complexities suggested in https://gist.github.com/karthik/a472880b07fa7ce4039a1cb5e96adec4 (after talking it over with Daniella).

- We get a Markdown template from the curators that people will need to fill out, we don't need to create an editor.
- [ ] Give information on the file upload page about readme and link to the markdown template, possibly also link to an online Markdown editor.
- [ ] Require that the readme file is in markdown format rather than any other for submission.
- [ ] I suppose there are bonus points if there is some way to validate the Markdown file itself if it's trivial to do.

**Comment by ryscher** [comment]
I think it's a great idea to require more detailed information, but I don't understand the proposed solution. Why burden the user with downloading a template and filling it out, when we already have a web form where we could just add the required fields? We can easily auto-generate our own Markdown file if we have the needed information.

**Comment by sfisher** [comment]
Hi @mariapraetzellis and @ryscher  We started enforcing README (any kind allowed like text files) a while back, but don't require READMEs for datasets created before December 12, 2021.

We did this to avoid what some people felt was a burden of having valid data meeting stricter requirements (users, curators) when they edited something.  This may be related to a ticket I saw in the backlog (allowing curators to bypass having valid data if they edit something).

Any thoughts?  Are we grandfathering people in so they don't need to make their data better if they go and edit it again or loosening enforcement by dates or something else?

**Comment by mariapraetzellis** [comment]
@sfisher I don't think we should enforce these new requirements for READMEs on existing datasets, only going forward. So, I think we are good to grandfather existing datasets that users may be editing or modifying and not requiring them to modify their READMEs.

**Comment by ryscher** [comment]
Agreed. If we applied these requirements to older datasets, we would annoy authors every time they created new versions. So it would be best to allow older datasets to keep less stringent restrictions.

**Comment by mariapraetzellis** [comment]
After consulting with curators, we added a line to the README template corresponding to the dataset file description. The updated template is [here](https://hackmd.io/0LR75OTLSrKfRe9qlcuzKg?both). @sfisher

**Comment by sfisher** [comment]
Thanks for the updates.  I've updated my pull request with the updated document and put it up on hackmd.io with a read-only version for people to start from.

**Comment by AmeliaCulpP1819** [comment]
Yay for README template available on the site! 
Curators discussed new language on 'Upload Files' page in our meeting today ("We require that you include a [README.md](https://datadryad.org/docs/README.md) file based on our template in order to provide key information for understanding and reuse of your data.") -- there is some concern that at this initial stage we may not be ready to require this template, but rather require a README (as we already do) and offer this template as a suggestion for the time being. Suggest changing the language to something like "We require that you include a README file in order to provide key information for understanding and reuse of your data. Please see our  template(https://datadryad.org/docs/README.md) for guidance."
